msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SignInEmailPasswordTest.kt

```diff
@@ -38,95 +41,70 @@ import org.junit.Ignore
 import org.junit.Test
 import org.mockito.Mockito
 import org.mockito.kotlin.spy
+import org.robolectric.RuntimeEnvironment.application
 
 class SignInEmailPasswordTest : NativeAuthPublicClientApplicationAbstractTest() {
 
-    override val defaultConfigType = ConfigType.SIGN_IN_PASSWORD
+    lateinit var application: INativeAuthPublicClientApplication
+    lateinit var config: NativeAuthTestConfig.Config
+
+    private val defaultConfigType = ConfigType.SIGN_IN_PASSWORD
+    private val defaultChallengeTypes = listOf("password", "oob")
 
     /**
      * Use valid email and password to get token.
      * (hero scenario 15, use case 1.2.1, Test case 37)
      */
     @Test
-    fun testSuccess() = runTest {
-        val username = config.email
-        val password = getSafePassword()
-        val result = application.signIn(username, password.toCharArray())
-        Assert.assertTrue(result is SignInResult.Complete)
+    fun testSuccess() {
+        config = getConfig(defaultConfigType)
+        application = setupPCA(config, defaultChallengeTypes)
+
+        runBlocking {
+            val username = config.email
+            val password = getSafePassword()
+            val result = application.signIn(username, password.toCharArray())
+            Assert.assertTrue(result is SignInResult.Complete)
+        }
     }
 
     /**
      * Use invalid email address to receive a "user not found" error.
      * (use case 1.2.2, Test case 38)
      */
     @Test
-    fun testErrorIsUserNotFound() = runTest {
-        val username = config.email
-        val password = getSafePassword()
-        // Turn an existing username to a non-existing username
-        val alteredUsername = username.replace("@", "1234@")
-        val result = application.signIn(alteredUsername, password.toCharArray())
-        Assert.assertTrue(result is SignInError)
-        Assert.assertTrue((result as SignInError).isUserNotFound())
+    fun testErrorIsUserNotFound() {
+        config = getConfig(defaultConfigType)
+        application = setupPCA(config, defaultChallengeTypes)
+
+        runBlocking {
+            val username = config.email
+            val password = getSafePassword()
+            // Turn an existing username to a non-existing username
+            val alteredUsername = username.replace("@", "1234@")
```

Comment:
Nit: Is it possible to use the `INVALID_EMAIL` from `NativeAuthPublicClientApplicationAbstractTest` to keep consistency with other tests? (For example, it is used in `SignUpEmailOTPTest.testErrorInvalidEmailFormat()`)

Same in line 184 with the `alteredPassword`.

Reply:
Done.